From 1bda52f80471260bcc7391f4e6919effedfc88d8 Mon Sep 17 00:00:00 2001
Message-Id: <1bda52f80471260bcc7391f4e6919effedfc88d8.1650802191.git.geliang.tang@suse.com>
From: Colin Ian King <colin.king@canonical.com>
Date: Sun, 15 Aug 2021 22:39:50 +0100
Subject: [PATCH] bpf, tests: Fix spelling mistake "shoft" -> "shift"
Git-commit: 1bda52f80471260bcc7391f4e6919effedfc88d8
Patch-mainline: v5.15-rc1
References: bsn#127
Modified-by-SEL: No


There is a spelling mistake in a literal string. Fix it.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Link: https://lore.kernel.org/bpf/20210815213950.47751-1-colin.king@canonical.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 lib/test_bpf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/test_bpf.c b/lib/test_bpf.c
index 44d8197bbffb..77fe6fde56c5 100644
--- a/lib/test_bpf.c
+++ b/lib/test_bpf.c
@@ -5163,7 +5163,7 @@ static struct bpf_test tests[] = {
 		{ { 0, -1 } }
 	},
 	{
-		"ALU64_ARSH_K: Zero shoft",
+		"ALU64_ARSH_K: Zero shift",
 		.u.insns_int = {
 			BPF_LD_IMM64(R0, 0x8123456789abcdefLL),
 			BPF_ALU64_IMM(BPF_ARSH, R0, 0),
-- 
2.26.2

